tests/kernel/gen_isr_table/testcase.yaml

```diff
@@ -20,7 +20,6 @@ tests:
   arch.interrupt.gen_isr_table.arm_mainline: &arm-mainline
     platform_allow: qemu_cortex_m3
     platform_exclude:
-      - stmf103_mini
```

Comment:
Actual board name is `stm32f103_mini`. Since this section excludes all boards based on the same SoC, it could be wise to keep it, as it probably won't pass the test if any one runs it (not us as this is not a ST board).
Up to you

Reply:
well, it builds and probably was there for a reason that does not exist anymore, so we can always re-add it.

Reply:
ok, I should probably complete the clean up for the similar other boards.